AUDIT-64 Use device information from database

drtyh2o opened the pull request:
## 📚 Purpose
There are scenarios where the device information (specifically the vehicle_id) in the cache gets out of sync with the information in the database. This seems to happen on occasion when a provider updates a device's vehicle_id. There is a separate effort to diagnose that issue, but in the meantime, this fix will always use device information from the database which we know to be accurate.

## 👌 Resolves:
- [x] 🐛 AUDIT-64 Searching for vehicles with updated vehicle_ids returns the correct vehicle but with the old vehicle_id

## 📦 Impacts:
- [x] mds-audit

avatarneil reviewed (approved):
May want to add a `//FIXME this should be investigated and changed back. We should assume the cache has the most up-to-date information, in a future version the db may lag behind.` or something

BigIdeas reviewed (approved):
Thanks for digging into this

drtyh2o commented:
> May want to add a `//FIXME this should be investigated and changed back. We should assume the cache has the most up-to-date information, in a future version the db may lag behind.` or something

I added a comment, but this sort of logic should actually be part of an Agency service for getting the latest information about a device/vehicle and not part of the Audit API at all.
